README.md

```diff
@@ -52,12 +52,75 @@ React.render(<TestComponent />, document.getElementById('content'));
 Optional props:
 
    - ```onContentChange```, function to be called on each content change (as the property name says!)
+   - ```styles```, see [Styling](#styling) below
 
 You can also listen to content changes on the editor. If you are using Reflux, by listening to the changes on ```MarkdownEditorContentStore```.
 To be able to do so, just ```require('react-markdown-editor').MarkdownEditorContentStore;```
 
 ## Dependencies
 You can modify the styles directly by modifying the styles declared in ```dist/MarkdownEditor.js```. The pre-existing styles assume that you are using Bootstrap and Font Awesome.
+
+## Styling<a name="styling"></a>
+```<MarkdownEditor />``` is styled using React's inline styling and can be styled by supplying a ```styles``` prop. The following keys are used to style each component, below are their default values:
```

Comment:
Do you have an example of a customized editor? A screenshot would be enough to put in the README if you are ok with it!